webpack/prod.js

```diff
@@ -1,27 +1,38 @@
-const merge = require("webpack-merge");
+const { merge } = require("webpack-merge");
 const path = require("path");
 const base = require("./base");
 const TerserPlugin = require("terser-webpack-plugin");
+const { CleanWebpackPlugin } = require("clean-webpack-plugin");
 
 module.exports = merge(base, {
   mode: "production",
   output: {
-    filename: "bundle.min.js"
+    filename: "bundle.min.js",
+  },
+  output: {
+    path: path.resolve(__dirname, "../dist/"),
+    filename: "bundle.min.js",
+    publicPath: "/",
   },
   devtool: false,
   performance: {
     maxEntrypointSize: 900000,
-    maxAssetSize: 900000
+    maxAssetSize: 900000,
   },
   optimization: {
     minimizer: [
       new TerserPlugin({
         terserOptions: {
           output: {
-            comments: false
-          }
-        }
-      })
-    ]
-  }
+            comments: false,
+          },
+        },
+      }),
+    ],
+  },
+  plugins: [
+    new CleanWebpackPlugin({
```

Comment:
Hey! In WebPack 5 there is a [clean option](https://webpack.js.org/guides/output-management/#cleaning-up-the-dist-folder) which removes the need for the `CleanWebpackPlugin` dependency.